modules/@apostrophecms/admin-bar/ui/apos/apps/force-refresh.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * If the delivered page has a logged-out content, but a cookie say "Hey, I'm logged in",
+ * we force-refresh the page to bypass the cache, in order to get the logged-in content.
+ */
+export default () => {
+  const isLoggedInPageContent = document.body.getAttribute('data-apos-user-logged-in');
+  console.log('isLoggedInPageContent', isLoggedInPageContent);
+
+  if (isLoggedInPageContent) {
+    return;
+  }
+
+  if (apos.util.getCookie('loggedIn')) {
```

Comment:
This cookie name should be namespaced, `aposLoggedIn` would be good. Usually we namespace cookies to the shortname to avoid confusion in dev since "localhost" is always the host there, but in this case the worst thing that can happen is a one-time extra refresh in a dev environment if you switch between testing two sites on localhost.

Reply:
noted, you're right!